packages/worker/README.md

```diff
@@ -0,0 +1,7 @@
+# @bugsnag/worker
+
+This package contains the worker implementation of the Bugsnag notifier for JavaScript.
```

Comment:
Is "worker" a specific enough term?  If this plugin works for [Web Workers](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Using_web_workers) in general then IMO we should say "Web Worker" and if it's specifically for Service Workers we should say "Service Worker".

As an aside, if it does work for Web Workers, do you have a feeling for 

* how that works?
* whether it's useful?

Reply:
TBC I'm talking about the description rather than the package name I think that is fine.

Reply:
the test fixture is using a worker, as opposed to a service worker. so I'm happy to change it to "web worker / service worker"

it should work in much the same way between web/service workers and should also handle uncaught errors in a web worker.